### PR TITLE
docs fixes

### PR DESCRIPTION
**Pull request by dornbirndevelops**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -1 +1,2 @@
 from crate.theme.rtd.conf.crate_python import *
+html_favicon = None
```
Comment (jodok) [inline docs/conf.py]: you should always add a newline at the end of the file :) (you can also configure your editor to do this automatically)

Comment (chaudum) [inline docs/conf.py]: @jodok there is already a newline character at the end of the line
python should not have a blank line at the end of the file (pep8 W391)

```console
$ pep8 docs/conf.py
docs/conf.py:4:1: W391 blank line at end of file
```

Comment (jodok) [inline docs/conf.py]: what does that symbol mean? 
image

Comment (chaudum) [inline docs/conf.py]: @jodok is right, the newline character is missing

Comment (dornbirndevelops) [inline docs/conf.py]: I'm sry that rule was new to me. I'll fix it.

**Pull request merged**